posts/2014-02-10-io-js-and-a-nodejs-foundation.md

```diff
@@ -0,0 +1,45 @@
+#io.js en een Node.js stichting
+#### De huidige staat van verzoening
+
+Vorige week nodigde Scott Hammond, CEO van Joyent, het [io.js TC][iojstc]
+(Technisch Committee) uit voor een gesloten vergadering waarin hij zijn
+intentie uitte om een node.js stichting te beginnen en zijn verlangen om io.js
+terug te krijgen in het node.js project.
+
+In de afgelopen maanden is io.js gegroeid to 23 actieve kernleden,
+verschillende _working groups_, 27 lokalisatie teams en is in staat geweest om
+met regelmaat kwaliteitssoftware te releasen dankzij de ondersteuning van een
+exceptionele community. We hebben dit weten te bereiken door een open
```

Comment:
Ik vind community wel lekker klinken. Het is strict gesproken niet Nederlands, maar we mogen best een beetje punk zijn en Engelse woorden gebruiken vind ik. :metal: